src/src/AppBundle/Command/TestCommand.php

```diff
@@ -18,13 +18,27 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         /** @var LoggerInterface $logger */
         $logger = $this->getContainer()->get('logger');
-        $output->writeln('Test messages are sending to message queue now. Please press CTRL+C to break this process.');
-        while(true) {
-            $logger->error('Some error occurred while doing sometihng.', [
-                'some_value'    => 'ABC',
-                'another_value' => 1532,
-            ]);
-            usleep(500);
+        $output->writeln('Test messages are sending to message queue now. Please press CTRL+C to break t');
+        $i=2;
+        while($i>1) {            
+            if($i%2==0){
+                $r=random_int(-1000, 0);
+                $logger->error('Some error occurred while doing sometihng.', [
+                            'some_value'    => 'ABC '.$r,
+                            'another_value' => $r,
+                        ]);
+                        usleep(500);
+            }else{ 
+                $r=random_int(-1000, 0);
+                $logger->warning('Some warning occurred while doing sometihng.', [
+                            'some_value'    => 'ABC '.$r,
+                            'another_value' => $r,
+                        ]);
+                        usleep(500);
+            }
+            $i++;
```

Comment:
Thank you so much for your contribution. Also, I'm so glad that it helped you. Shall we make it a bit simpler? Logger allows you to pass the log type with a parameter through `log()` method. So maybe you can do something like the following to make it simple:
```php
while (true) {
  $r = random_int(-1000, 0);
  $message = 'Some warning occurred while doing sometihng.';
  $context = ['some_value' => 'ABC '.$r, 'another_value' => $r];
  $logType = ( microtime(true) * 10000 % 2 == 0) ? 'error' : 'warning';
  $logger->log($logType, $message, $context);
  usleep(500);
} 
```
Thanks!